benchmarks/CPU/instr/main.c

```diff
@@ -97,69 +96,64 @@ int main(int argc, char *argv[])
     unsigned cpu_mask = 0xffffffff;
     int opt;
 
-    /* timer stuff*/ 
+    // timer stuff
     long period_ms = 0;
     long utilization_ratio = 100;
 
     while ((opt = getopt(argc, argv, "l:m:p:u:")) != -1) {
         switch (opt) {
-        case 'l':
-            loops_per_print = xstrtol(optarg, "-l");
-            break;
-        case 'm':
-            cpu_mask = xstrtol(optarg, "-m");
-            break;
-        case 'p':
-            period_ms = xstrtol(optarg, "-p");
-            break;
-        case 'u':
-            utilization_ratio = xstrtol(optarg, "-u");
-            break;
-        default: /* '?' */
-            fprintf(stderr, "Usage: %s [-l loops ] [-m cpu_mask]\n",
-                    argv[0]);
-            exit(1);
+            case 'l':
+                loops_per_print = xstrtol(optarg, "-l");
+                break;
+            case 'm':
+                cpu_mask = xstrtol(optarg, "-m");
+                break;
+            case 'p':
+                period_ms = xstrtol(optarg, "-p");
+                break;
+            case 'u':
+                utilization_ratio = xstrtol(optarg, "-u");
+                break;
+            default: /* '?' */
+                fprintf(stderr, "Usage: %s [-l loops ] [-m cpu_mask]\n", argv[0]);
+                exit(1);
         }
     }
 
-    if(utilization_ratio < 0 || utilization_ratio > 100)
+    if (utilization_ratio < 0 || utilization_ratio > 100) {
         errx(1, "%s", "utilization ratio is not in the interval [0,100]!");
+    }
 
-    if (utilization_ratio == 0)
-        idle_thread = 1;
-    if (utilization_ratio == 100)
-        idle_thread = 0;
-    else
-        idle_thread = 0;
+    idle_thread = utilization_ratio == 0 ? 1 : 0;
 
     #ifdef WITH_DEMOS
-		printf("Running benchmark with DEmOS support enabled.\n");
-		if (demos_init() != 0) {
-			errx(1, "%s", "Could not initialize DEmOS scheduler - are you running the program within DEmOS?");
-		}
-	#else
-		printf("Running benchmark without DEmOS support.\n");
-	#endif
+        printf("Running benchmark with DEmOS support enabled.\n");
+        if (demos_init() != 0) {
+            errx(1, "%s", "Could not initialize DEmOS scheduler - are you running the program within DEmOS?");
+        }
+    #else
+        printf("Running benchmark without DEmOS support.\n");
+    #endif
 
     for (int i = 0; i < num_proc; i++) {
         pthread_attr_t attr;
         cpu_set_t cpuset;
         pthread_t tid;
 
-        if ((cpu_mask & (1U << i)) == 0)
-            continue;
+        if ((cpu_mask & (1U << i)) == 0) continue;
```

Comment:
No.

Reply:
Current clang-format config formats the `continue;` to new line, which is imo 1) waste of screen space, 2) dangerous, as multiline ifs without braces are calling for trouble when someone adds statements after the if. I'll keep it the way it originally was for now, but for new code, I frankly don't think it's good style.

Reply:
Don't discuss uselessly about style. It is an existing project which already uses some style. If we change the style to according to each contributor, it'd be a mess.